README.md

```diff
@@ -5,8 +5,29 @@ Reading instructions:
 *   under each subheading, the first code block is SAS and the second code block is the R translation;
 *   ctrl+F is your friend.
 
+`data` step
+-----------
+
+#### concatenate datasets ####
+
+```SAS
+data concatenated;
+    set mydata1 mydata2;
+run;
+```
+
+```r
+# Note that if a factor variable in the two dataframes has different
+# levels, then rbind will take the union of these levels
+# while row_bind will coerce the variable to string.
+concatenated = rbind(mydata1, mydata2)
+
+# or with dplyr
+library(dplyr)
+concatenated = row_bind(mydata1, mydata2)
+```
 
-Save data to disk
+#### Save data to disk ####
 -----------------
```

Comment:
Remove the underlining hyphens "`-`", which indicate a 2nd level header (`<h2>` in HTML) and conflicts with the 4th level header `####` symbol.